features/other_b/skin.feature

```diff
@@ -433,3 +433,19 @@ Feature: creating and editing skins
     And I should see "only screen and (max-width: 42em), only screen and (max-width: 62em)"
   When I press "Use"
   Then the page should have a skin with the media query "only screen and (max-width: 42em), only screen and (max-width: 62em)"
+
+  Scenario: A user should be able to choose a temporary skin
+  Given basic skins
+    And the approved public skin "public skin" with css "#title { text-decoration: blink;}"
+   Then I am logged in as an admin
```

Comment:
Adding coverage for the interface is a good idea, yep! I think the thing to do is use the shortened version I suggested for users and then set up a test for "Scenario: Admin can cache a public skin" specifically for testing caching the skin through the interface, and maybe even one for "Scenario: Admin can add a public skin to the chooser"